.github/workflows/cron-update-rust.yml

```diff
@@ -10,6 +10,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
```

Comment:
Do we need this new GH Action ?

Although it's suggested in the `peter-evans/create-pull-request`, can't we just follow the same approach as rust-bitcoin ([here](https://github.com/rust-bitcoin/rust-bitcoin/blob/de8c70ccfd59d0ed0758936a5fc7a40afa266ac6/.github/workflows/cron-semi-weekly-update-nightly.yml#L30-L41)) and use a new token ?